src/main/java/com/sybit/airtable/exception/HttpResponseExceptionHandler.java

```diff
@@ -22,11 +24,15 @@ public class HttpResponseExceptionHandler {
     public static void onResponse(HttpResponse response) throws AirtableException {
 
         final Integer statusCode = response.getStatus();
-        String message = convertStreamToString(response.getRawBody());
```

Comment:
RawBody is no longer publicly exposed, but getParsingError seems to do the right thing here instead.